### PR TITLE
feat(KUI-1266): upgrade to KTH Style 10

### PR DESCRIPTION
**Pull request by karlandindrakryggen**
This PR upgrades KTH Style to version 10 and includes some more cleanup and fixes:

- Upgrade to KTH style packages 
  - Includes major cleanup of CSS
  - Adding same shared concept as in public web apps with copied files in `_shared.scss` and `components-shared` folder 
- CSS for preview has been extracted to a separate SCSS file and more or less been synced with style from `kurs-pm-web`
- Fixed bug with sticky feature of tabs
- Upgrade of other dependencies (minor/patch)
- Hardcoded toolbar URL are replaced with env variable
- Favicon path is fixed

https://kth-se.atlassian.net/browse/KUI-1266

### CODE REVIEW

File: public/css/node-web.scss [hunk 1]
```diff
@@ -1,962 +1,234 @@
...
-  @media (min-width: 992px) {
-    #mainMenu.col {
-      max-width: 245px;
-    }
-    .progress-title {
-      display: flex !important;
+    @media (max-width: 991px) {
+      // hacks to disable sticky section menu in smaller screens
+      // TODO: a better solution might be to refactor the sticky feature completly and use position:sticky instead of react-sticky package
+      // hack 1: override position set by react-sticky
```
Comment (karlandindrakryggen) [inline public/css/node-web.scss:114]: This is part of a bugfix for the sticky feature. My TODO comment are a suggestion for the future, but not really something we need to prioritize now.

`react-sticky` is in maintenance mode has a recommendation to stop using it on its Readme:

_"It is highly recommended that you begin transitioning to another sticky library to ensure better support and sustainability. "_

https://github.com/captivationsoftware/react-sticky

**Comment by karlandindrakryggen** [comment]
Related PRs:
- kursinfo-admin-web: https://github.com/KTH/kursinfo-admin-web/pull/273
- kurs-pm-admin-web: https://github.com/KTH/kurs-pm-admin-web/pull/276

**Review by amirhossein-haerian — APPROVED** [review]

File: .azure/ref.parameters.json [hunk 2]
```diff
@@ -43,7 +43,8 @@
         "STDOUT_ENABLED": "true",
         "SESSION_USE_REDIS": "true",
         "UG_REST_AUTH_API_TOKEN_URI": "https://login.ref.ug.kth.se/adfs/",
-        "UG_REST_API_URI": "https://integral-api.sys.kth.se/test/ug"
+        "UG_REST_API_URI": "https://integral-api.sys.kth.se/test/ug",
+        "TOOLBAR_URL": "https://www-r.referens.sys.kth.se/social/toolbar/widget.js"
       }
```
[the remark this answers is not in the document]
Comment (karlandindrakryggen) [reply .azure/ref.parameters.json:48]: The toolbar (commonly known as "KPM" or "Personliga menyn") is a widget in the top of the page that all KTH websites should have. It is developed by an other team (E-lärande) and the way it is loaded into our page is that we include the script. 

Social is another web developed by E-lärande, so I guess social and KPM is connected in some way and that is way init file is hosted on /social.

**Review by axelbjo — APPROVED** [review]

File: .azure/ref.parameters.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (amirhossein-haerian) [reply .azure/ref.parameters.json:48]: Thanks for the clear explanation, now I have a better understanding of it 🚀

**Review by belanglos — APPROVED** [review]
Hästjobb! Bra jobbat! 🤘

**Pull request merged**